components/builder-worker/src/runner/logger.rs

```diff
@@ -13,26 +13,60 @@
 // limitations under the License.
 
 use std::fs::File;
-use std::io::Write;
+use std::io::{BufRead, BufReader, Write};
+use std::process;
 
 use super::workspace::Workspace;
 
+const EOL_MARKER: &'static str = "\n";
```

Comment:
We control the output of the log so this shouldn't become a problem on Windows - we're not going to use their line endings anyway